metadata_tools/geometry_support.py

```
@@ -275,38 +498,37 @@ def add(self, qualifier, *,
+            tiles = []
 
-        # Get the column decsriptions
+        # Get the column descriptions
         column_descs = self.dicts[qualifier]
         if name:
             column_descs = column_descs[name]
 
         # Prepare the rows
-        rows = Record._prep_row(self.prefixes, self.backplane, self.blocker, column_descs,
+        rows, overrides = self._prep_row(self.prefixes, self.backplane, self.blocker, column_descs,
                                 primary=self.primary, target=target,
```

Flake8 (7.3.0)

[error] continuation line under-indented for visual indent

(E128)